kstatus/README.md

```diff
@@ -0,0 +1,116 @@
+# kstatus
+
+kstatus provides tools for checking the status of Kubernetes resources. The primary use case is around the pull-apply loop
+where a set of resources are updated/created with `apply` and we want to know when the changes has either fully
+been reconciled in a cluster or something is causing the process to stall.
+
+## Concepts
+
+This effort has several goals, some with a shorter timeline than others. Initially, we want to provide
+a library that makes it easier to decide when changes to a set of resources have been reconciled in a cluster.
+Since the built-in kubernetes types doesn't provide this information directly, we will initially rely on rules
```

Comment:
Since the built-in... -> To support types that do not yet publish status information

initially rely on rules specific for each type -> fallback on type specific rules